src/validation/update.beneficial.owner.indiviual.review.validation.ts

```diff
@@ -0,0 +1,44 @@
+import { body } from "express-validator";
```

Comment:
Is this not all a lot of unnecessary copy and pasted validations that already exist?
Can you not use ...validator.updateBeneficialOwnerIndividual

Reply:
No we cannot because of dob

Reply:
hmm that's annoying .. still, it would be better to define these validations once in the codebase and re-use. So existing BeneficialOwner validations and your new one could import for e.g 
...name_validations
...nationality_validations
..sanctions_list_validations

etc.

Reply:
I don't want to hold up anything with my comments if it works @olaych  :)
but for the next sprint and the upcoming review tickets for the other BO types we can refactor these validations so they are only defined once and then re-used everywhere, instead of copying them more times, @stephanie-ch is that ok?